Overview.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+Learn a little bit more about Flare's firey APIs
+
+## LocalFirestore
+
+Flare's Local Firestore command allows for the reading, writing, and deleting of documents to be seamless and easy to use.
+There are a couple functions you can call to edit a document. SetDocument, GetDocumentOnce, and println() are all functions used to
+read, write, and display data values.
+
+```kotlin
+suspend fun main() {
+
+    @Serializable
+    data class Dog( //Class created to store id, name, age and string that is being written to the document
+        @SerialName(DocId)
+        val id: String,
+        val name: String,
+        val age: Int
+    )
+
+    //Calling Local Firestore to read and write in a document
+    val firestore = LocalFirestore()
+    val writeDog = Dog("a3f6awe9", "Max", Int.MAX_VALUE)
+    firestore.setDocument("dogs/${writeDog.id}", writeDog) //Adds the writeDog variable to the document
+    // specified by the path given
+    val readDog: Dog = firestore.getDocumentOnce("dogs/${writeDog.id}").data() //Reads what is in the document
+    // (Once again specified by the path given)
+    println(readDog)
+}
+```
+
+## LocalAuth
+
+LocalAuth is used for the authentification and identification of who someone is. It can help with creating an account,
+signing into an account, signing out, and deleting the account. Creating an account locally by calling a couple functions
+makes the authentication experience much more efficient for developers.
+
+```kotlin
+suspend fun main() {
+
+    @Serializable
+    data class Dog( //Class created to store id, name, age and string that is being written to the document
+        @SerialName(DocId)
+        val id: String,
+        val name: String,
+        val age: Int
+    )
+
+    //Sign in, sign out, sign back in, delete account
+    val auth = LocalAuth()
+
+    val method: AuthMethod = AuthMethod.EmailPassword(email = "johnnyappleseed@pear.com", password = "johnny123")
+    auth.signIn(method) //signs in using the method variable that is given - contains email and password of the user
+    auth.signOut()
+    auth.signIn(method) //signs in once again using the method variable
+    auth.currentUser!!.delete()   //deletes the account if the user is registered and logged in
+}
+```
+
+## LocalStorage
+
+LocalStorage is used to manage files locally. By calling the LocalStorage() library you are able to
+use a couple functions such as EncodeToByteArray(), PutBytes(), GetBytes(), DecodeToString(), and DeleteFile().

```

Comment:
You don't need to name the used methods.... We'll have better source docs reference in the future 🙏